SQL/database_changelog.txt

```diff
@@ -1,3 +1,23 @@
+20 May 2017, by Jordie0608
+
+Created table `round` to replace tracking of the datapoints 'round_start', 'round_end', 'server_ip', 'game_mode', 'round_end_results', 'end_error', 'end_proper', 'emergency_shuttle', 'map_name' and 'station_renames' in the `feedback` table.
+Once created this table is populated with rows from the `feedback` table.
+
+START TRANSACTION;
+CREATE TABLE `feedback`.`round` (`id` INT(11) NOT NULL AUTO_INCREMENT, `start_datetime` DATETIME NOT NULL, `end_datetime` DATETIME NULL, `server_ip` INT(10) UNSIGNED NOT NULL, `server_port` SMALLINT(5) UNSIGNED NOT NULL, `game_mode` VARCHAR(32) NULL, `game_mode_result` VARCHAR(64) NULL, `end_state` VARCHAR(64) NULL, `shuttle_name` VARCHAR(64) NULL, `map_name` VARCHAR(32) NULL, `station_name` VARCHAR(80) NULL, PRIMARY KEY (`id`));
+INSERT INTO `feedback`.`round`
```

Comment:
So while I didn't get any duplicate errors I didn't test all the data nor at once since it took my computer ages to process and I got bored once the queries started timing out.
Nevertheless I don't think it's a problem with the query so I would have to guess there some (probably very old) garbage data around somewhere.